internal/command/command.go

```diff
@@ -75,29 +75,29 @@ var CmdGenerate = &Command{
 	Name:  "generate",
 	Short: "Generate a new client library",
 	Run: func(ctx context.Context) error {
-		// tmpRoot is a newly-created working directory under /tmp
-		// We do any cloning or copying under there.
-		tmpRoot, err := createTmpWorkingRoot(time.Now())
-		if err != nil {
-			return err
-		}
-
 		if flagAPIPath == "" {
 			return fmt.Errorf("-api-path is not provided")
 		}
 		if !supportedLanguages[flagLanguage] {
 			return fmt.Errorf("invalid -language flag specified: %q", flagLanguage)
 		}
-
-		var apiRoot string
 		if flagAPIRoot == "" {
-			repo, err := cloneGoogleapis(ctx, tmpRoot)
```

Comment:
No more automatic cloning?

Reply:
Not for now - the "spec" (the doc) says that it's required. But I do want to do that in a follow-up. (I figured I'd implement what was written first.)

Reply:
(I was confused - this is generate, not configure.) It's required in the proposed spec, and I think it makes sense to keep it that way. The use cases for generate are generally about testing a speculative API that you already have locally - it doesn't make much sense (IMO) to generate a pristine clone.